doc/configuration.rst

```diff
@@ -634,6 +634,34 @@ therefore auto-cleanup is important.
 Used by:
   - `DockerDriver`_
 
+LXAIOBusPIO
+~~~~~~~~~~~
+
+A :any:`LXAIOBusPIO` resource describes an a single PIO pin on a LXAIOBusNode.
+
+.. code-block:: yaml
+
+   LXAIOBusPIO:
+     host: localhost:8080
+     node: IOMux-00000003
+     pin: OUT0
+     invert: False
+
+The example uses a lxa-iobus-server running on localhost:8080, with node
+IOMux-00000003 and pin OUT0.
+
+- host (str): hostname with port of the lxa-io-bus server
```

Comment:
Inconsistency, mostly. We can separate it here as well, since this way we can enforce a port number through the attrs type.

Reply:
Sounds good.

Reply:
Eh, all of the resource combine the port into the device. The port attribute is only used for serialports (because they are serialports), in the networkservice. In case of the networkservice, this should probably be fixed and converted to the SSHService anyway. So I'd like to keep it as is.